modules/ROOT/pages/deployment/general/general-info.adoc

```diff
@@ -188,6 +170,81 @@ IMPORTANT: The command line option `--force-overwrite` is only intended for deve
 
 To reinitialize Infinite Scale, you have to delete your config and your data and start from scratch.
 
+== Define the Infinite Scale Data Path
+
+Because Infinite Scale does not use a database for storing information like users, groups, spaces, internal data, etc., it saves all this data to a permanent file location. This location is also used for storing user-generated data and must be a supported filesystem as described in xref:prerequisites/prerequisites.adoc#filesystems-and-shared-storage[Filesystems and Shared Storage].
+
+The environment variable used to define this path is `OCIS_BASE_DATA_PATH`.
+
+The following rules apply:
+
+* If you do NOT define this environment variable, the following applies:
+** The base path is by default `$HOME/.ocis/` after performing a _manual setup_.
+** The base path is by default `/var/lib/ocis` inside the container when using the _container setup_.
+* The directory must exist and the user used for Infinite Scale must have full access and permissions.
+
+NOTE: You must set this environment variable to a valid path when using the manual installation having a system user for Infinite Scale, because a system user has no logon and therefore no home directory!
+
+WARNING: The location must exclusively be used by Infinite Scale. Writing into this location not using Infinite Scale is strictly discouraged to avoid any unexpected behaviour. 
```

Comment:
```suggestion
WARNING: The location must be used by Infinite Scale exclusively. Writing into this location not using Infinite Scale is discouraged to avoid any unexpected behavior. 
```